packages/component/src/Attachment/Text/private/CodeContent.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Fragment, memo, ReactNode, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import createHighlighter from './shiki';
+
+type Props = Readonly<{
+  children?: ReactNode | undefined;
+  className?: string | undefined;
+  code: string;
+  language?: string | undefined;
+  title: string;
+}>;
+
+const highlighterPromise = createHighlighter();
```

Comment:
Should this be lazy so it won't impact page load?

Reply:
My concern is it then affects 1st modal opening. We already have everything loaded, maybe we could take some time to set it up as well. But I'm ready to change if you think it should be the other way around

Reply:
Maybe as a ShikiProvider later?

Reply:
Yea, will make it a provider on the next iteration!